src/main/java/org/spongepowered/common/data/manipulator/immutable/item/ImmutableSpongeAuthorData.java

```diff
@@ -37,7 +37,7 @@
 
 public class ImmutableSpongeAuthorData extends AbstractImmutableSingleData<Text, ImmutableAuthorData, AuthorData> implements ImmutableAuthorData {
 
-    final ImmutableSpongeValue<Text> author;
+    private final ImmutableSpongeValue<Text> author;
```

Comment:
Make sure this isn't used anywhere that would create a synthetic accessor with this change. I don't think many people set their IDEs to warn about those but most things set to package-private is to avoid the extra synthetic.